### ISSUE TITLE
Cannot find module 'lodash'

### ISSUE THREAD

**Issue by ghost** [tcoopman/elm-css-webpack-loader#6]
Hi, I tried to run the example but I got the error `Cannot find module 'lodash'`:

![image](https://cloud.githubusercontent.com/assets/83847/19454118/24f1ae10-94b8-11e6-8aaf-fb43f3bf9176.png)

Do you know why? Thanks!

**Comment by erlandsona** [comment]
Me too...

**Comment by erlandsona** [comment]
Just realized this is fixed by adding the
     noParse: /.elm$/,
option to the module definition in webpack.config.js

**Comment by erlandsona** [comment]
That said, then we're left with the issue that I'm still stuck on... `Uncaught ReferenceError: require is not defined    app.js:8166`

**Comment by ghost** [comment]
Same error here @erlandsona :/

**Comment by mezuzza** [comment]
+1 Same error as @erlandsona.

**Comment by bredikhin** [comment]
FWIW, running `npm i` in both module root and examples folder fixes the issue.

**Comment by ahstro** [comment]
Anyone looking into this? For some reason, it seems to work perfectly in the example, even when I extract the example folder and install `elm-css-webpack-loader` as a module from npm, but I can't get it to work with my work's setup.

If I use `loader: "style!css!elm-css", I get the aforementioned `require is not defined` runtime error, but if I try to use it with `ExtractTextPlugin`, I instead get this error during build:

```
    ERROR in ./elm/Stylesheets.elm
    Module build failed: Error: Cannot find module '../node_modules/css-loader/lib/css-base.js'
```

**Comment by jiwhiz** [comment]
Please take a look at my PR #12 , I upgraded the example to Elm 0.18 and Webpack 2.2.

For a more advanced example, see my [elm-bootstrap-webpack-starter](https://github.com/jiwhiz/elm-bootstrap-webpack-starter) project.

**Comment by erlandsona** [comment]
@ahstro I'm having the same issue as you are now... 
```
    ERROR in ./elm/Stylesheets.elm
    Module build failed: Error: Cannot find module '../node_modules/css-loader/lib/css-base.js'
```
Have you found a fix since you posted?

**Comment by erlandsona** [comment]
I tried @jiwhiz's pr but I'm still getting it...

**Comment by erlandsona** [comment]
I found the cause! Turns out I was using a relative path in my css for `prop "background" "url(images/stairs.jpg) center 27% no-repeat"` and that was causing an error in the css compilation which at the top of the error read the same as the Cannot find module, probably because if the elm doesn't compile the loaders throw their own errors. That makes sense I guess?

**Comment by ahstro** [comment]
@erlandsona I don't have any relative path's in my CSS, I barely have any CSS at all.
I think the error is related to changing path somehow. I tried hooking into css-loader [here](https://github.com/webpack-contrib/css-loader/blob/master/lib/loader.js#L111) and adding a `console.log(process.cwd())` before the require, and for every piece of CSS it shows `/` except for that generated by `elm-css-webpack-loader`, for which it shows `/home/ahstro/path/to/package.json-directory`. Don't have time to debug it further now, but hopefully someone can figure out what's happening

**Comment by ahstro** [comment]
Okay, I definitely haven't found out what's going on - if anything, I'm more confused than ever - but I've figured out a workaround.

First off, this doesn't seem to have anything to do with `elm-css-webpack-loader`. I'm getting the same error just by creating a `Stylesheet.elm` and including `/\.elm$/` in my current CSS loading setup, without even installing any Elm related dependencies.

After some playing around, I realized that the `.elm` suffix, for whatever reason, broke things. A valid CSS file with `.el` worked, so did `.e` and `.foo`, but as soon as I changed the import to `.elm`, BIG RED ERROR.

My workaround is to name the file `Stylesheet.elm.css` instead, and just change the loader to reflect that. Seems to work flawlessly for me so far, and I've been able to apply styles to the website I'm working on, so not sure what else could go wrong.
